### PR TITLE
docs: fix small error

### PR DESCRIPTION
**Pull request by 0xflotus**
I only fixed a small error in docs.

# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by Yukang-Lian — APPROVED** [review]
LGTM

**Comment by Yukang-Lian** [comment]
There are confilcts in this PR, could you plz resolve it? Thanks a lot!

**Review by hf200012 — APPROVED** [review]
LGTM

File: docs/en/docs/data-table/data-model.md [hunk 1]
```diff
@@ -493,7 +493,7 @@ Therefore, when there are frequent count (*) queries in the business, we recomme
 
 Add a count column and import the data with the column value **equal to 1**. The result of `select count (*) from table;`is equivalent to `select sum (count) from table;` The query efficiency of the latter is much higher than that of the former. However, this method also has limitations, that is, users need to guarantee that they will not import rows with the same AGGREGATE KEY column repeatedly. Otherwise, `select sum (count) from table;`can only express the number of rows originally imported, not the semantics of `select count (*) from table;`
 
-Another way is to **change the aggregation type of the count column above to REPLACE, and still weigh 1**. Then`select sum (count) from table;` and `select count (*) from table;` the results will be consistent. And in this way, there is no restriction on importing duplicate rows.
+Another way is to **change the aggregation type of the count column above to REPLACE, and still weigh 1**. Then `select sum (count) from table;` and `select count (*) from table;` the results will be consistent. And in this way, there is no restriction on importing duplicate rows.
```
Comment (Zmaul04) [inline docs/en/docs/data-table/data-model.md:496]: image**__

File: docs/en/docs/install/install-deploy.md [hunk 2]
```diff
@@ -226,7 +226,7 @@ See the `lower_case_table_names` section in [Variables](../../advanced/variables
 
 * BE webserver_port configuration
```
Comment (Zmaul04) [inline docs/en/docs/install/install-deploy.md:227]: src/Essentials/src/Connectivity/Connectivity.uwp.cs

File: docs/en/docs/install/install-deploy.md [hunk 3]
```diff
@@ -226,7 +226,7 @@ See the `lower_case_table_names` section in [Variables](../../advanced/variables
...
 
-	If the BE componet is installed in hadoop cluster, you need to change the configuration `webserver_port=8040`  in be.conf to avoid port conflicts.
+	If the BE component is installed in hadoop cluster, you need to change configuration `webserver_port=8040` to avoid port used.
```
Comment (Zmaul04) [inline docs/en/docs/install/install-deploy.md:229]: @0xflotus

File: docs/en/docs/install/install-deploy.md [hunk 4]
```diff
@@ -226,7 +226,7 @@ See the `lower_case_table_names` section in [Variables](../../advanced/variables
...
 
 * Set JAVA_HOME environment variable
```
Comment (Zmaul04) [inline docs/en/docs/install/install-deploy.md:231]: JavaScript

File: docs/en/docs/install/install-deploy.md (hunk 2 above)
Comment (Zmaul04) [inline docs/en/docs/install/install-deploy.md:227]: - [ ]

File: docs/en/docs/data-table/data-model.md (hunk 1 above)
Comment (Zmaul04) [inline docs/en/docs/data-table/data-model.md:496]:
```

```

**Review by Zmaul04 — COMMENTED** [review]
- [ ]

**Review by Zmaul04 — COMMENTED** [review]
- __

**Pull request merged**